### PR TITLE
Update pakckages and cleanup

### PR DESCRIPTION
**Pull request by ccruz2886**
- In order to reduce the install package time, I removed the unneeded dependencies.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,15 +4,13 @@
...
+    "react": "^16.1.0",
     "react-dom": "^16.0.0",
     "react-redux": "^5.0.6",
     "react-router-dom": "^4.2.2",
-    "react-scripts": "1.0.14",
+    "react-scripts": "1.0.17",
     "redux": "^3.7.2",
-    "styled-components": "^2.2.1",
-    "yarm": "^0.4.0",
-    "yarn": "^1.3.2"
```
Comment (ccruz2886) [inline package.json:15]: I think this should be global.

File: package.json [hunk 2]
```diff
@@ -4,15 +4,13 @@
...
-    "react": "^16.0.0",
+    "react": "^16.1.0",
     "react-dom": "^16.0.0",
     "react-redux": "^5.0.6",
     "react-router-dom": "^4.2.2",
-    "react-scripts": "1.0.14",
+    "react-scripts": "1.0.17",
     "redux": "^3.7.2",
-    "styled-components": "^2.2.1",
-    "yarm": "^0.4.0",
```
Comment (ccruz2886) [inline package.json:14]: I didn't found if this dependency is being used. 🙏 please confirm

Comment (darieldejesus) [reply package.json:14]: Yep, you're right.

**Comment by darieldejesus** [comment]
Looks good.

**Pull request merged**